codec/src/main/java/io/netty/handler/codec/compression/ZstdDecoder.java

```diff
@@ -34,15 +37,15 @@ public final class ZstdDecoder extends ByteToMessageDecoder {
     {
         try {
             Zstd.ensureAvailability();
+            outCapacity = ZstdBufferDecompressingStream.recommendedTargetBufferSize();
```

Comment:
@luben I guess this is the best I can do ? Or is there a ways to try to see if the target buffer is big enough and if not try again with a bigger one ?

Reply:
Yes, I think this is the best you can do.

Reply:
Not really - we pass that buffer down to the Zstd native library.